static-assets/components/cstudio-common/resources/en/base.js

```diff
@@ -847,6 +847,7 @@ CStudioAuthoring.Messages.registerBundle('contentTypes', 'en', {
   enableSearchExisting: 'Enable Search Existing',
   useSearch: 'Use Search',
   allowEditWithoutWarning: 'Allow Edit Without Warning',
+  replaceAccent: 'Replace Accent and Diacritic',
```

Comment:
'Allow Accents and Diacritics'

Reply:
Sumer,
Talking with Phil, 
The feature does not allow the unicode characters. Instead it converst them to latin chars.
I suggest we relabel this to "Convert accent/diacritics to latin characters”

Reply:
I updated the text as a suggestion from Russ. Thanks!